src/component/mxgraph/renderer/style-utils.ts

```diff
@@ -17,15 +17,29 @@
 import { mxCell } from 'mxgraph'; // for types
 
 /**
- * Compute the all class names associated to a given bpmn element kind in an hyphen case form.
+ * Compute the all class names associated to a cell in an hyphen case form.
  *
- * @param bpmnElementKind the string representation of a BPMN element kind i.e {@link ShapeBpmnElementKind} and {@link FlowKind}.
+ * @param cell the `mxCell` related to the BPMN element.
  * @param isLabel the boolean that indicates if class must be computed for label.
  * @internal
  */
-export function computeAllBpmnClassNames(bpmnElementKind: string, isLabel: boolean): string[] {
+export function computeAllBpmnClassNamesOfCell(cell: mxCell, isLabel: boolean): string[] {
```

Comment:
ℹ️ I have introduced this function because it was weird to call a chain of 2 functions from this file in  `ShapeConfigurator`.
It will also prepare implementation of #928: we will need to look at the whole mxCell style to retrieve the bpmn semantic informaiton